lib/honeycomb/client.rb

```diff
@@ -1,12 +1,18 @@
 # frozen_string_literal: true
 
+require "forwardable"
 require "honeycomb/beeline/version"
 require "honeycomb/configuration"
 require "honeycomb/context"
 
 module Honeycomb
   # The Honeycomb Beeline client
   class Client
+    extend Forwardable
+
+    def_delegators :@context, :current_span, :current_trace
```

Comment:
Noting here that it might be worth using these delegators across the `Honeycomb::Client` code instead of `context.current_span` all over the place. 💅 ✨